.github/workflows/tests.yml

```diff
@@ -199,9 +193,9 @@ jobs:
       ###
       # Setup Ruby - this needs to match the version in the Gemfile
       - name: Set up Ruby
-        uses: ruby/setup-ruby@a699edbce608a2c128dedad88e3b6a0e28687b3c
+        uses: ruby/setup-ruby@b161cb92f0e2f28d992f04f38fff204d84132c47
         with:
-          ruby-version: 2.6.6
+          ruby-version: .ruby-version
```

Comment:
This is nice.

Reply:
Discovered this earlier today. Got to throw away a bunch of hacks which did the same thing.

Reply:
That's great.